### ISSUE TITLE
Relationship comments don't show up in a serialized document

### ISSUE THREAD

**Issue by armintaenzertng** [spdx/spdx-java-jackson-store#40]
```
ISerializableModelStore modelStore = new MultiFormatStore(new InMemSpdxStore(), MultiFormatStore.Format.XML);
String documentUri = "https://some.namespace";
ModelCopyManager copyManager = new ModelCopyManager();

SpdxDocument document = SpdxModelFactory.createSpdxDocument(modelStore, documentUri, copyManager);

document.setSpecVersion(Version.TWO_POINT_THREE_VERSION);
document.setName("SPDX-tool-test");
Checksum sha1Checksum = Checksum.create(modelStore, documentUri, ChecksumAlgorithm.SHA1, "d6a770ba38583ed4bb4525bd96e50461655d2758");
AnyLicenseInfo concludedLicense = LicenseInfoFactory.parseSPDXLicenseString("LGPL-2.0-only");

SpdxFile fileA = document.createSpdxFile("SPDXRef-fileA", "./fileA.c", concludedLicense,
                List.of(), "Copyright 2022 some person", sha1Checksum)
        .build();

SpdxFile fileB = document.createSpdxFile("SPDXRef-fileB", "./fileB.c", concludedLicense,
                List.of(), "Copyright 2022 some person", sha1Checksum)
        .build();

document.getDocumentDescribes().add(fileA);

fileB.addRelationship(
        document.createRelationship(
                fileA, RelationshipType.DEPENDS_ON, "some relationship comment"));

modelStore.serialize(documentUri, new FileOutputStream("output.xml"));
```
The above code generates an SPDX document with two files that are connected via a `DEPENDS_ON` relationship with a comment. Serializing this in the last line yields an output file that contains the following lines regarding relationship:
```
<relationships>
  <spdxElementId>SPDXRef-fileB</spdxElementId>
  <relationshipType>DEPENDS_ON</relationshipType>
  <relatedSpdxElement>SPDXRef-fileA</relatedSpdxElement>
</relationships>
```
i.e. the relationship comment is missing. The same happens in all other formats of the `MultiFormatStore` and when using the `TagValueStore` via
```
ISerializableModelStore modelStore = new TagValueStore(new MultiFormatStore(new InMemSpdxStore(), MultiFormatStore.Format.XML));
```
On the other hand, the `RdfStore` prints out relationship comments just fine.

**Comment by goneall** [comment]
PR #39 should have this one fixed

**Comment by goneall** [comment]
Resolved with PR #39
